views/admin/tpl/email/module_support_email.tpl

```diff
@@ -0,0 +1,182 @@
+[{assign var="shop"      value=$oEmailView->getShop()}]
+[{assign var="oViewConf" value=$oEmailView->getViewConfig()}]
+[{assign var="userinfo"  value=$oEmailView->getUser()}]
+[{assign var="header_template" value="`$shopTemplateDir`email/html/header.tpl"}]
+[{assign var="footer_template" value="`$shopTemplateDir`email/html/footer.tpl"}]
+
+[{include file=$header_template title=$email_data.subject}]
+
+<table border="0" width="100%" cellspacing="10" cellpadding="2" bgcolor="#FFFFFF" class="column">
+  <tr>
+    <td class="text-pad" style="white-space:nowrap;">
+      <b style="white-space:nowrap;">[{oxmultilang ident="module_support_email_from"}]</b>
```

Comment:
I think you don't need to specify `white-space` on the inner element. It should be inherited automatically. Actually, if you want none of the words to break in the whole table, why not specify `white-space:nowrap;` on the table instead?